### ISSUE TITLE
Banner images and menu do not display on basic install

### ISSUE THREAD

**Issue by finnlewis** [localgovdrupal/localgov_subsites#7]
Since porting new functionality from Lambeth, the banner image and sidebar menu for subsites do not display out of the box.

I am working on this and will push a branch for testing.

**Comment by Adnan-cds** [comment]
Yeah, I had this issue :)  I remember manually placing the banner and navigation blocks.  The 2 block config files bundled with this module are for the localgov_theme theme.  For everyone else, you either need some instructions in the README or an install hook.

**Comment by finnlewis** [comment]
resolved with https://github.com/localgovdrupal/localgov_subsites/issues/7

**Comment by markconroy** [comment]
This issue looks fixed. Closing.

PR: https://github.com/localgovdrupal/localgov_subsites/pull/8
